nifi-nar-bundles/nifi-graph-bundle/nifi-graph-processors/src/test/java/org/apache/nifi/processors/graph/TestExecuteGraphQuery.java

```diff
@@ -41,9 +41,10 @@
 public class TestExecuteGraphQuery {
     protected TestRunner runner;
 
-    @Rule public MockitoRule mockitoRule = MockitoJUnit.rule();
+    @Rule
+    public MockitoRule mockitoRule = MockitoJUnit.rule();
```

Comment:
The `@Rule` annotation is a JUnit 4 feature that is implemented differently in JUnit 5. In this particular test class, it looks like the annotation and the `MockitoRule` variable can be removed